app/components/screen/screen.tsx

```diff
@@ -41,7 +41,7 @@ function ScreenWithoutScrolling(props: ScreenProps) {
     >
       <StatusBar
         barStyle={props.statusBar || statusBarContent}
-        backgroundColor={props.backgroundColor}
+        backgroundColor={styles.background.color}
```

Comment:
you can actually use `colors.white` instead, `colors` being destructured from 

```
  const {
    theme: { colors },
  } = useTheme()
```

then you can probably refactored this style by removing it entirely:

```
const useStyles = makeStyles(({ colors }) => ({
  background: {
    color: colors.white,
  },
}))
```